src/gui/wxgui/CMakeLists.txt

```diff
@@ -118,6 +116,13 @@ add_library(CemuWxGui STATIC
   wxHelper.h
 )
 
+if (ENABLE_BLUEZ)
+  add_library(CemuWxGui STATIC
```

Comment:
This should be `target_sources(CemuWxGui PRIVATE`
Currently it breaks the build because add_library tries to create a new library with the same name